CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 ## TBD
 ### Changed
 * Changed recursion logic to prevent unnecessary calls.
+* Changed the name of the database file for h2 to reflect the name of the schema in the configuration file (instead of having `housekeeping.mv.db` for any schema name).
```

Comment:
I think Changed is already said by the heading. So, all the statements should start from just what is being changed.
eg. 

Changed
* the name of the database....

Reply:
Same for all other statements that are changed.